src/lib/pipeline_configuration.py

```diff
@@ -30,9 +30,13 @@ class CodeSchemes(object):
     CHALLENGES = _open_scheme("challenges.json")
 
     SOMALIA_OPERATOR = _open_scheme("somalia_operator.json")
+    SOMALIA_REGION = _open_scheme("somalia_region.json")
```

Comment:
I would put these with the district schemes rather than the operator

Reply:
Updated thanks